### PR TITLE
Release Note for 1.7.0

### PR DESCRIPTION
**Pull request by chunseoklee**
Release note for 1.7.0 is added

### CODE REVIEW

File: docs/release/1.7/release-note-1.7.0.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+### New command line interface for user interface consistancy
+
+- one-import: imports conventional models files to circle
+   - one-import-tf: imports TensorFlow model to circle
+   - one-import-tflite: imports TensorFlow lite model to circle
+- one-optimize: circle optimize command
+- one-quantize: circle quantize command
+   - supports float32 to uint8, layer wise (for Conv series)
+- one-pack: package command
+- one-prepare-venv: prepares python virtual environment for import TensorFlow model
```
Comment (llFreetimell) [inline docs/release/1.7/release-note-1.7.0.md:26]:
```suggestion
- one-prepare-venv: prepares python virtual environment for importing TensorFlow model
```

File: docs/release/1.7/release-note-1.7.0.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  
+## ONE Compiler
+
+### Compiler supports more operations
+
+- MatrixDiag, MatrixSetDiag, ReverseSequence, ReverseV2, SegmentSum, SelectV2, SparseToDense, Where
+
+### New command line interface for user interface consistancy
+
+- one-import: imports conventional models files to circle
```
Comment (llFreetimell) [inline docs/release/1.7/release-note-1.7.0.md:19]:
```suggestion
- one-import: imports conventional model files to circle
```

File: docs/release/1.7/release-note-1.7.0.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+
```
Comment (llFreetimell) [inline docs/release/1.7/release-note-1.7.0.md:1]:
```suggestion
```

**Comment by seanshpark** [comment]
There is no signed off :)

File: docs/release/1.7/release-note-1.7.0.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+ ## Feature Highlights
```
Comment (llFreetimell) [inline docs/release/1.7/release-note-1.7.0.md:1]:
```suggestion
## Feature Highlights
```

**Comment by llFreetimell** [comment]
@git-bot-sec help

**Comment by llFreetimell** [comment]
@nnfw-bot help

**Comment by llFreetimell** [comment]
@nnfw-bot test all

**Comment by chunseoklee** [comment]
@Samsung/nnfw PTAL

**Pull request merged**